pipelines/imputation/testing/ImputationBeagleEmpty.wdl

```diff
@@ -16,7 +16,6 @@ workflow ImputationBeagle {
         String reference_panel_path_prefix
         String genetic_maps_path
         String output_basename
-        Boolean split_output_to_single_sample = false
```

Comment:
this is cleanup unrelated to this PR - this input is no longer in the pipeline wdl